task1.js

```diff
@@ -0,0 +1,7 @@
+let str1 = prompt("Word");
+let str2 = prompt("Word");
+let str3 = prompt("Word");
```

Comment:
It will be better to use ```const``` in current case. 
Use human-readable variable names in next lessons, pls. It could decrement your mark(